modules/backlogs/config/locales/en.yml

```diff
@@ -175,7 +171,7 @@ en:
   rb_label_copy_tasks_open: "Open"
   rb_label_link_to_original: "Include link to original story"
 
-  remaining_hours: "remaining hours"
+  remaining_hours: "remaining work"
 
   required_burn_rate_hours: "required burn rate (hours)"
   required_burn_rate_points: "required burn rate (points)"
```

Comment:
What is the reason for the translations being in here when the implementation is moved to the core?

Reply:
It was forgotten in a previous commit.